config.go

```
@@ -97,31 +97,24 @@ func getVolumeFilePath(volumeName string) string {
 	return filepath.Join(volumePath, volumeCfg)
 }
 
-func getVolumeNames(driver BackupStoreDriver) ([]string, error) {
+// getVolumeNames returns all volume names based on the folders on the backupstore
+func getVolumeNames(driver BackupStoreDriver) []string {
 	names := []string{}
-
 	volumePathBase := filepath.Join(backupstoreBase, VOLUME_DIRECTORY)
-	lv1Dirs, err := driver.List(volumePathBase)
-	// Directory doesn't exist
-	if err != nil {
-		return names, nil
-	}
+	lv1Dirs, _ := driver.List(volumePathBase)
 	for _, lv1 := range lv1Dirs {
 		lv1Path := filepath.Join(volumePathBase, lv1)
-		lv2Dirs, err := driver.List(lv1Path)
-		if err != nil {
-			return nil, err
-		}
+		lv2Dirs, _ := driver.List(lv1Path)
```

Ignore the error can result in hard to debug issues. We can put those errors in log instead without error out.